### PR TITLE
[repo] Prepare release Exporter.Geneva-1.9.0-rc.1

### PR DESCRIPTION
**Pull request by opentelemetrybot**
Note: This PR was opened automatically by the [prepare release workflow](https://github.com/open-telemetry/opentelemetry-dotnet-contrib/actions/workflows/prepare-release.yml).

## Changes

* CHANGELOG files updated for projects being released.

### CODE REVIEW

File: src/OpenTelemetry.Exporter.Geneva/CHANGELOG.md [hunk 1]
```diff
@@ -2,6 +2,13 @@
 
 ## Unreleased
 
+## 1.9.0-rc.1
+
+Released 2024-Jun-10
+
+* Update OpenTelemetry SDK version to `1.9.0-rc.1`.
+  ([#1834](https://github.com/open-telemetry/opentelemetry-dotnet-contrib/pull/1869))
```
Comment (CodeBlanch) [inline src/OpenTelemetry.Exporter.Geneva/CHANGELOG.md:10]: Has 1834 in the text but links to 1869

Comment (vishweshbankwar) [inline src/OpenTelemetry.Exporter.Geneva/CHANGELOG.md:10]:
```suggestion
  ([#1869](https://github.com/open-telemetry/opentelemetry-dotnet-contrib/pull/1869))
```

Comment (vishweshbankwar) [reply src/OpenTelemetry.Exporter.Geneva/CHANGELOG.md]: good catch. fixed.

**Review by rajkumar-rangaraj — APPROVED** [review]

**Review by Kielek — CHANGES_REQUESTED** [review]
I do not think that the changelogis true, see https://github.com/open-telemetry/opentelemetry-dotnet-contrib/blob/6b604c4b2e030b91958a55da70d4dcc7842f496c/src/OpenTelemetry.Exporter.Geneva/OpenTelemetry.Exporter.Geneva.csproj#L18

Feel free to ignore blocking whe you fix/double checking

**Comment by vishweshbankwar** [comment]
> I do not think that the changelogis true, see
> 
> https://github.com/open-telemetry/opentelemetry-dotnet-contrib/blob/6b604c4b2e030b91958a55da70d4dcc7842f496c/src/OpenTelemetry.Exporter.Geneva/OpenTelemetry.Exporter.Geneva.csproj#L18
> 
> Feel free to ignore blocking when you fix/double checking

@Kielek - `OtelSdkVersion` actually points to `OpenTelemetryCoreLatestPrereleaseVersion`. See https://github.com/open-telemetry/opentelemetry-dotnet-contrib/blob/6b604c4b2e030b91958a55da70d4dcc7842f496c/src/OpenTelemetry.Exporter.Geneva/Common.GenevaExporter.props#L3

Will be working on removing this indirect dependency.

**Comment by vishweshbankwar** [comment]
I am going to hold on to this for now: would like to get the change added in Log Exporter to support https://github.com/open-telemetry/opentelemetry-dotnet/blob/main/src/OpenTelemetry/CHANGELOG.md#190-rc1. Will send a separate PR.

**Review by Kielek — APPROVED** [review]

File: src/OpenTelemetry.Exporter.Geneva/CHANGELOG.md [hunk 2]
```diff
@@ -1,6 +1,11 @@
 # Changelog
 
-## Unreleased
+## 1.9.0-rc.1
```
Comment (Kielek) [inline src/OpenTelemetry.Exporter.Geneva/CHANGELOG.md:3]:
```suggestion
## Unreleased

## 1.9.0-rc.1
```

**Comment by CodeBlanch** [comment]
Just FYI the automation didn't run when this was merged because after #1873 it looks for "[release]" in the PR title not "[repo]". I asked @vishweshbankwar to manually push the tag. Sorry @vishweshbankwar!

**Pull request merged**